slack/greeting

```diff
@@ -1,19 +1,19 @@
-Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+:wave: Hey {{eec4a322-082c-4247-8cdd-b67d5eeee6aa==user}}, Welcome to the JSON Schema Slack Server!
```

Comment:
I don't know what's responsible for interpolating these variables, but is the change here from `{{Joiner}}` to this random looking UUID correct?

Reply:
I would say no. I THINK when I copied it here originally, I had to manually replace those.
It is Slack that's responsible for those ID thinggy in that case.

Reply:
Fixed.